SUPPORT.md

```diff
@@ -0,0 +1,22 @@
+# Support for deploying and using KEDA
+
+Welcome to KEDA! We use GitHub for tracking bugs and feature requests but here are some pointers to get help.
+
+## Documentation
+
+* [User Documentation](https://keda.sh/)
+* [Troubleshooting Guide](https://keda.sh/docs/2.0/troubleshooting/)
+* [FAQ](https://keda.sh/docs/latest/faq/)
+
+## Slack
+
+* [Slack](https://kubernetes.slack.com) ([registration](http://slack.k8s.io)):
+The `#keda` channel is usually the place where people offer support.
+
+## Kubernetes Support
+
+The supported window of Kubernetes versions with KEDA is known as "N-2": (N (Latest release) - 2 (minor versions)).
```

Comment:
I am not sure if this is true, or if we even can say something like this. Currently KEDA can run on k8s >= 1.16 and I don't see us bringing any new features, that will break support on that version.

And we don't actively test against Kubernetes version (N-2), we run our tests on 1.18 only.

Reply:
The reason why I've added it here is mainly for us having to comply with at least N-2 support and not use new features that are available in version N. This helps customers understand what to expect and relates to https://github.com/kedacore/charts/issues/104 for example.

Makes sense or not?

Reply:
OKay :)